html/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Let's not have an `html` folder with just one file. Instead, just put `index.html` up one folder.